ballerina-tests/tests/resources/expected_results/null_value_for_nullable_input_object_with_error_added_to_context.json

```
@@ -0,0 +1,11 @@
+{
+    "errors": [
+        {
+            "message": "Data not found",
+            "path": ["name"]
+        }
+    ],
+    "data" : {
+        "name" : null
+    }
+}
```

Check the IDE settings whether it adds an empty line at the end of each file.

```suggestion
}

```